src/app/components/MainContent.tsx

```diff
@@ -31,22 +31,40 @@ const ContentStyles = styled(({ textSize, ...props }) => <DynamicContentStyles {
 
 // tslint:disable-next-line:variable-name
 const MainContent = React.forwardRef<HTMLDivElement, React.PropsWithChildren<Props>>(
-  ({book, children, className, ...props}, ref) => <Consumer>
-    {({registerMainContent}) => <main
-      ref={mergeRefs(ref, registerMainContent)}
-      className={className}
-      tabIndex={-1}
-    >
-      <ContentStyles
-        id={MAIN_CONTENT_ID}
-        book={book}
+  ({book, children, className, ...props}, forwardedRef) => {
+    const initialLoad = React.useRef(true);
+    const ref = React.useRef<HTMLDivElement>(null);
+
+    React.useImperativeHandle(forwardedRef, () => ref.current as HTMLDivElement);
+
+    React.useEffect(
+      () => {
+        if (initialLoad.current) {
+          initialLoad.current = false;
+          document?.getElementById('root')?.querySelector('a')?.focus();
```

Comment:
the task is to focus the main container right? why are you focusing the first anchor here?

Reply:
Because the first load of the page should focus on the Skip to content link. Only subsequent changes should focus on changed content. I guess this is the function that `resetTabIndex` was doing previously, but it did it on every content change. Setting focus to document.body might be more clear (or, equivalently, calling blur on activeElement)

Reply:
Maybe we just want this always to happen where previously we had it resetting the focus.
Intuitively, that sounds likely. `scrollToTopOrHashManager` is called by PageComponent on DidMount and DidUpdate. They are differentiated by whether there is a `previous`.